src/screens/Teachers.js

```diff
@@ -12,10 +13,30 @@ export default class TeachersScreen extends React.Component {
         this.state = {
             isLoading: true,
             data: props.route.params.teachers ?? [],
-            user: props.route.params.user ?? {}
+            user: props.route.params.user ?? {},
+            searchQuery: '',
         }
+        this.clearSearch = this.clearSearch.bind(this);
       }
 
+    handleSearch = (text) => {

```

Comment:
You don't need to bind this function?

Reply:
I'm really unclear *why* this doesn't need to be binded, but it does seem to function properly without the bind.

Reply:
Digged a little deeper, and the arrow function syntax does the same thing as binding. So we have two different syntaxes that achieve the same thing here.